recipes/gdal/post_3.5.0/conanfile.py

```diff
@@ -0,0 +1,490 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches
+from conan.tools.cmake import CMakeToolchain, cmake_layout
+from conans import CMake, tools
```

Comment:
It's odd. You should use either legacy cmake generator, or new CMakeToolchain with a layout, not both.